draft-ietf-quic-recovery.md

```diff
@@ -1028,9 +1029,9 @@ min_rtt:
 
 max_ack_delay:
 : The maximum amount of time by which the receiver intends to delay
-  acknowledgments, in milliseconds.  The actual ack_delay in a
-  received ACK frame may be larger due to late timers, reordering,
-  or lost ACKs.
+  acknowledgments. The value is initialized to 0, and updated when the peer's
+  transport parameters are received. The actual ACK Delay in a received ACK
+  frame may be larger due to late timers, reordering, or lost ACKs.
```

Comment:
How about clarifying tha the value is zero for "crypto packets".

Some reasons:
* endpoints are expected to send ACKs immediately for crypto packets, even after TP.max_ack_delay is known.
* Retransmissions of 0-RTT packets should respect max_ack_delay obtained in the previous connection.

Reply:
I think this text needs to clarify that the value is updated to the default 25ms if TPs are received with the max_ack_delay setting.

Reply:
Instead of crypto packets, I'd suggest packets in the ApplicationData packet number space?

Reply:
@ianswett Done.